### ISSUE TITLE
Figure out which email address to use for account creation

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#6609]
> What application are you using?

Other / Not Sure

> Describe the problem.

Ramona A has returned to work at ATD after working at the Convention Center.  CTM has given her an alias email address that includes a "2".  What email address should I have entered in Knack when she was hired.  Thanks, Linda 

> How soon do you need this?

Soon — This week 

> Requested By
Linda E.

Request ID: DTS21-102235

**Comment by SurbhiBakshi** [comment]
@scottreedy had mentioned that CTM is creating new emails for people moving between departments, is there a way to get back to the original email without a number?  Teams messaging Scott.

**Comment by SurbhiBakshi** [comment]
Emailed  Linda to let her know I will get back to her when I know more.

**Comment by SurbhiBakshi** [comment]
Linda has used the original email address in the HR application and Ramona A has a service request with CTM to get her original email address. I have asked Ramona to keep Linda in the loop in case Linda has to update the HR application.

**Comment by dianamartin** [comment]
Christina called me to help with Ramona being able to log into the HR app so she can fill out her TLC form. I checked the system and she's in there as first.last name and not with the "2" at the end. I had her test if she could log in, but she said that the system just looped her back to through the app. 

I added the "2" and had her try again to log in, she was successful. I'm having her communicate to Linda that she was able to get into the application and for the time being it's "2" until CTM can resolve her email situation. Ramona will reach back out to Linda whenever the email addresses needs to be changed back to not have the "2" at the end.
